envs/doom/multiplayer/tests/test_vizdoom_multiplayer.py

```diff
@@ -51,7 +51,7 @@ def test_doom_multiagent(self):
         self.doom_multiagent(self.make_standard_dm, worker_index=0)
 
     def test_doom_multiagent_parallel(self):
-        num_workers = 16
+        num_workers = 1
```

Comment:
The whole point of this test is running VizDoom in parallel. If this does not work in Travis (why wouldn't it by the way??)
then you should create two tests and skip one in Travis somehow. I do not approve this change, unless you explain why it is needed.